src/main/java/org/prebid/server/privacy/gdpr/vendorlist/VendorListServiceV2.java

```diff
@@ -26,6 +26,7 @@
 
     public VendorListServiceV2(String cacheDir,
                                String endpointTemplate,
+                               boolean deprecated,
```

Comment:
Change arg order as in parent `org.prebid.server.privacy.gdpr.vendorlist.VendorListService` class.